vignettes/model.Rmd

```diff
@@ -88,68 +88,93 @@ We model the expected number of infections/latent notifications ($\lambda^l$) us
 
 ## Default model
 
-In our default model we consider the delay distribution to follow a discretised log-normal as follows,
+In our default model, we consider the delay to follow a discretised log-normal distribution, specified as
 
 \begin{align}
   p_{g,t,d} &\sim \text{LogNormal} \left(\mu^d, \sigma^d \right) \\
   \mu^d &\sim \text{Normal} \left(0, 1 \right) \\
   \sigma^d &\sim \text{Half-Normal} \left(0, 1 \right)
 \end{align}
 
+Here, $p_{g,t,d}$ is the probability that a case in group $g$ with reference time $t$ is reported with delay $d$. 
+
 ## Generalised model
 
-We generalise this model in order to support a range of delay distributions and reporting processes following the approach of Günther et al.[@gunther2021] we estimate the delay distribution ($p_{g,t,d}$) using a discrete-time logistic hazard model
+We generalise this model in order to support a range of delay distributions as well as effects of the reporting process on the delay. Following the approach of Günther et al.[@gunther2021] and others, we parameterise the delay probability ($p_{g,t,d}$) via a discrete-time hazard model, i.e.
+
+\begin{equation}
+  p_{g,t,0} = h_{g,t,0},\ p_{g,t,d} = \left(1 -\sum^{d-1}_{d^{\prime}=0} p_{g,t,d^{\prime}} \right) \times h_{g,t,d},
+\end{equation}
+
+where
+
 $$
-h_{g,t,d} =\text{P} \left(\text{delay}=d|\text{delay} \geq d, W_{g,t,d}\right)
+h_{g,t,d} =\text{P} \left(\text{delay}=d|\text{delay} \geq d, W_{g,t,d}\right),
 $$
-but we extend this model to decompose $W_{g,t,d}$ into 3 components: hazard derived from a parametric delay distribution ($\gamma_{g,t,d}$) dependent on covariates at the time of reference, hazard not derived from a parametric distribution ($\delta_{g,t,d}$) dependent on covariates at the time of reference, and hazard dependent on covariates referenced to the time of report ($\epsilon_{g,t,d}$).
+is the so-called reporting hazard. For a case in group $g$ with reference time $t$, the hazard $h_{g,t,d}$ states the probability of being reported with delay $d$, given that the case is not reported earlier. The hazard depends on a design matrix $W_{g,t,d}$, which encodes a baseline delay distribution and covariates that affect the reporting delay. We extend the model of Günther et al. by decomposing the hazard into three components,
+
+  - **Parametric baseline hazard $\gamma_{g,t,d}$**: hazard derived from a parametric delay distribution, with parameters depending on covariates at the time of reference
+  - **Non-parametric reference time effect $\delta_{g,t,d}$**: effect on the hazard that depends on covariates at the time of reference
+  - **Non-parametric report time effect $\epsilon_{g,t,d}$**: effect on the hazard that depends on covariates at the time of report
+
+Each component adds to the overall hazard through a regression with logit link
```

Comment:
With a logit link?

Reply:
you mean "through a regression with a logit link"?